src/components/admin/AdminProduct.vue

```diff
@@ -51,89 +51,163 @@ const handleDeletion = () => {
 </script>
 
 <template>
-  <div v-if="headingFlag" class="admin-product-list__heading admin-product-heading text-sm">
-    <span v-for="category in categoryList" class="admin-product-heading__category">
-      {{ category }}
-    </span>
-  </div>
+    <div v-if="headingFlag" class="admin-product-list__heading admin-product-heading text-sm">
+        <div v-for="(category, index) in categoryList" :key="index" class="admin-product-heading__category">
+            {{ category }}
+        </div>
+    </div>
+
+    <div v-else class="admin-product-list__card admin-product-card">
+        <img class="admin-product-card__image" :src="imageUrl" :alt="name" />
 
-  <div v-else class="admin-product-list__card admin-product-card">
-    <img class="admin-product-card__image" :src="imageUrl" :alt="name" />
+        <div v-for="(category, index) of [code, name, `${price} USD`, stock, discount ? discount : 'N/A']"
+            :class="`text-sm admin-product-card__${['code', 'name', 'price', 'stock', 'discount'][index]}`">
+            {{ category }}
+        </div>
 
-    <span
-      v-for="(category, index) of [code, name, `${price} USD`, stock, discount ? discount : 'N/A']"
-      :class="`text-sm admin-product-card__${['code', 'name', 'price', 'stock', 'discount'][index]}`"
-    >
-      {{ category }}
-    </span>
+        <div class="admin-product-card__button-wrapper">
+            <button @click="open" class="text-sm admin-product-card__patch-button">
+                <EditIcon />
+            </button>
 
-    <div class="admin-product-card__button-wrapper">
-      <button @click="open" class="text-sm admin-product-card__patch-button">
-        <EditIcon width="2.4rem" height="2.4rem" />
-      </button>
+            <button @click="handleDeletion" class="text-sm admin-product-card__delete-button">
+                <TrashIcon />
+            </button>
+        </div>
 
-      <button @click="handleDeletion" class="text-sm admin-product-card__delete-button">
-        <TrashIcon width="2.4rem" height="2.4rem" />
-      </button>
+        <GenericToast v-if="deletionFlag" v-bind="toastPreset" />
```

Comment:
GenericToast as component is dead and should stay dead.

Reply:
Not my part